README.md

```diff
@@ -70,6 +70,33 @@ keys listed below. They can be defined by setting the configuration key in:
 |koji.hub.url|The Koji Hub URL to find builds|http://brewhub.localhost/brewhub|
 |koji.web.url|The Koji Web URL|http://brewweb.localhost/brew|
 |pnc.url|The PNC URL to find builds|http://pnc.localhost|
+|infinispan-mode|Define whether to use Infinispan in EMBEDDED (default) or REMOTE|EMBEDDED|
```

Comment:
Maybe a `.` instead of a `-`. I guess the idea is it's sort of a namespace. I'd also add some backticks around the code constants or you could not capitalize them and treat them as normal words.

Reply:
Sure